code/game/objects/items/storage/boxes.dm

```diff
@@ -136,6 +136,9 @@
 	if(HAS_TRAIT(SSstation, STATION_TRAIT_PREMIUM_INTERNALS))
 		new /obj/item/flashlight/flare(src)
 		new /obj/item/radio/off(src)
+		new /obj/item/gas_filter(src)
+	else
+		new /obj/item/gas_filter/damaged(src)
```

Comment:
Are we really adding more things to the survival box? Don't you just put this on the breath mask? Why not just put it in immediately?

Reply:
i'll remove it. This doesn't change or touch the breath masks in any way